examples/pytorch/BUILD

```diff
@@ -1,12 +1,114 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
+load("@rules_python//python:defs.bzl", "py_binary")
+load("@pip_deps//:requirements.bzl", "requirement")
+load("@jvolkman_rules_pycross//pycross:defs.bzl", "pycross_wheel_library")
```

Comment:
Why is the extra dependency necessary vs just using py_binary?

Reply:
Because `py_binary` requires dependencies returning `PyInfo` interface and `py_wheel` doesn't implement it.